src/main/java/org/codehaus/mojo/servicedocgen/descriptor/OperationDescriptor.java

```diff
@@ -116,7 +116,7 @@ public void setHttpMethod( String method )
      */
     public String getDescription()
     {
-        return this.description;
+        return this.description.replace("'", "\"");
```

Comment:
interesting. Why do we have to do this? If this is for escaping JSON we should properly escape (via backslash).
IMHO this is missplaced here. We should rather have a simple `EscapeHelper` class that would provide such logic via static `escapeJson` and `escapeHtml` methods and could be used inside the velocity template around all variables to escape. Then the escaping can be done format specific. E.g. in HTML instead `<` would need to be escaped.

Reply:
Actually JavaDoc is already HTML so no escaping needed for HTML...

Reply:
Yes, it is needed for escaping the JSON string. Moved the escaping into an helper class.